Godeps/_workspace/src/k8s.io/kubernetes/pkg/securitycontextconstraints/provider.go

```diff
@@ -228,20 +241,7 @@ func (s *simpleProvider) ValidateContainerSecurityContext(pod *api.Pod, containe
 		allErrs = append(allErrs, fielderrors.NewFieldInvalid("privileged", *sc.Privileged, "Privileged containers are not allowed"))
 	}
 
-	if sc.Capabilities != nil && len(sc.Capabilities.Add) > 0 {
-		for _, cap := range sc.Capabilities.Add {
-			found := false
-			for _, allowedCap := range s.scc.AllowedCapabilities {
-				if cap == allowedCap {
```

Comment:
we previously required capabilities match allowedCapabilities with exact case... is there a good reason to loosen that?